src/shared-components/accordion/style.ts

```diff
@@ -2,16 +2,26 @@ import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 import {
   ANIMATION,
-  BOX_SHADOWS,
   BREAKPOINTS,
+  BOX_SHADOWS,
   COLORS,
   SPACER,
 } from 'src/constants';
 
 const border = `1px solid ${COLORS.border}`;
 
+const getTopBorderRadius = (borderRadius: string) => ({
+  borderTopLeftRadius: borderRadius,
+  borderTopRightRadius: borderRadius,
+});
+
+const getBottomBorderRadius = (borderRadius: string) => ({
+  borderBottomLeftRadius: borderRadius,
+  borderBottomRightRadius: borderRadius,
+});
```

Comment:
I'm on the fence about these functions, since they seem to only give `borderRadius` two additional names without any modifications. It seems like it might be adding some unnecessary complexity, since `borderRadius` could be used in place of these to the same effect.

Reply:
The long-form properties are used because the `:first-of-type` and `:last-of-type` conflict when there is only one `<Accordion>` item. Using a single `border-radius` would override one or the other. 

I can remove the function (or rename suffix to `Radiuses`?) if it's unclear.

Reply:
@ZeMunchkin Opted to just remove the helpers entirely in https://github.com/curology/radiance-ui/pull/371/commits/ac26593096f3a93d5643702a403b503c80b458e4 🙂